common/buildings/wc_duchy_capital_buildings.txt

```diff
@@ -113,8 +113,8 @@ aerodrome_02 = {
 		modifier = {
 			factor = -20
 			NOT = {
-				has_unlocked_special_aerial_maa_trigger = yes
-				has_unlocked_special_massive_aerial_maa_trigger = yes
+				has_unlocked_culture_aerial_maa_trigger = yes
```

Comment:
It scopes to province, you should scope to culture of province by putting these triggers in `culture = { }`.